archive.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const args = process.argv.slice(2, process.argv.length);
+const { MArchive, MgArchive } = require('./lib/archive.js');
+
+if (args[0] === 'm') {
+  if (args[1] === 'put') {
+    MArchive.put(args.slice(2, args.length));
+    return;
```

Comment:
These `return` statements appear redundant. I would remove the return statements and add else statement instead. Example:
`
if (args[0] == 'm'){
   if (args[1]) === 'putt) {
     ...
   }
}
else if (args[1] === 'get' && args.length === 2){
   MArchive.get();
}
`

Reply:
Both are fine options, but I believe an early return solution is cleaner and nicer to work with (I've gone with the 1st option described here: https://stackoverflow.com/q/1820839/4699289).

Struggling to find a good article which compares these two solutions, this was the closest I could find: https://blog.timoxley.com/post/47041269194/avoid-else-return-early

Reply:
Interesting point Sam! I read the articles that you gave and see the value of the early return statement. Indicating that there is no more processing increases readability significantly. 

I think that the most readable approach is to have early return statements *and* the else statement. The else statement highlights to the reader the alternative flow paths. The early return statements highlights to the reader the flow end. This dual approach's greater readability outweighs the minor increase in duplication.

Reply:
I've gone for `if else` and `return` 👍